### PR TITLE
Cleanup docs for the release

### PR DESCRIPTION
**Pull request by KitHat**
Updated documentation to v.1.7.0

#### PR Checklist

- [x] Documentation

### CODE REVIEW

File: docs/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -5,7 +5,6 @@
 * Runtime Descriptions
 ** xref:runtime/xcm_executor.adoc[XCM Executor]
 * Pallet Specifications
-** xref:pallets/aura_ext.adoc[cumulus_aura_ext]
```
Comment (ozgunozerk) [inline docs/modules/ROOT/nav.adoc:8]: why is this removed?

**Review by ozgunozerk — APPROVED** [review]
seems good to me overall except the comment I made

File: docs/modules/ROOT/nav.adoc (hunk 1 above)
Comment (KitHat) [reply docs/modules/ROOT/nav.adoc:8]: This is a duplicate of the same link 2 lines lower
